integration/server.go

```diff
@@ -137,7 +137,7 @@ func (s *server) start() error {
 		if err != nil {
 			// Don't propagate 'signal: killed' error,
 			// since it's an expected behavior.
-			if !strings.Contains(err.Error(), "signal: killed") {
+			if !strings.Contains(err.Error(), "signal: killed") && errb.Len() > 0 {
```

Comment:
NIT, `errb` is a terrible name. Consider `stderr`
```suggestion
			if !strings.Contains(err.Error(), "signal: killed") && stderr.Len() > 0 {
```

Reply:
`stderr` is already taken by the `StdErrPipe()` of the cmd started. This is a buffer that buffers every byte read from `stderr` (since an `io.Reader` can only be read once). I'll rename it to `stderrBuf`.